server/src/LspServer.ts

```diff
@@ -137,6 +138,19 @@ export class LspServer {
     };
   }
 
+  onUncaughtException(error: Error, origin: 'uncaughtException' | 'unhandledRejection'): void {
+    console.log(`${new Date().toUTCString()} ${origin}:`, error.message);
+    console.log(error.stack);
```

Comment:
The same will be logged later in `sendTelemetry` ?

Reply:
Yes. I think it is not a big problem and we want to have no such situations at all.